generator/src/test/java/com/scottlogic/deg/generator/outputs/targets/FileOutputTargetTests.java

```diff
@@ -0,0 +1,86 @@
+package com.scottlogic.deg.generator.outputs.targets;
+
+import com.scottlogic.deg.common.profile.Profile;
+import com.scottlogic.deg.generator.outputs.formats.OutputFormat;
+import com.scottlogic.deg.generator.utils.FileUtils;
+import org.junit.jupiter.api.Test;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class FileOutputTargetTests {
+    //Create Mocks
+    private Path mockFilePath = mock(Path.class);
```

Comment:
Generally I prefer to follow the pattern of recreating my mocks afresh for every test in a setup method.

In the case of the tests you have at present, it doesn't make any difference.  However, in other situations - for example if the setup of a mock for one test could conflict with the setup of that mock for another test, or in situations where the Assert section of a test checks the state of a mock - recreating your mock objects avoids annoying and potentially intermittent bugs.